File: src/lib/FrameworkManager/sample/packages/ProjectPackage/core/EmptyModel.java
/*
 *  %modelName%Model
 *  自由に拡張可能です
 *
 *  Copyright (c) 2014年 saimushi. All rights reserved.
 */

package com.unicorn.model;

public class %modelName%Model extends %modelName%ModelBase {

	/**
	 * 短縮コンストラクタ
	 * @param argContext Contextが入っています
	 */
	public %modelName%ModelBase(Context argContext) {
		super(argContext, Constant.PROTOCOL, Constant.DOMAIN_NAME, Constant.URL_BASE,
				Constant.COOKIE_TOKEN_NAME, Constant.SESSION_CRYPT_KEY, Constant.SESSION_CRYPT_IV, Constant.TIMEOUT);
	}
}

File: src/lib/FrameworkManager/sample/packages/ProjectPackage/core/EmptyModel.m
//
//  %modelName%Model.m
//  自由に拡張可能です
//
//  Copyright (c) 2014年 saimushi. All rights reserved.
//

#import "common.h"
#import "%modelName%Model.h"

@implementation %modelName%Model

/* オーバーライド */
- (id)init;
{
    self = [super init:PROTOCOL :DOMAIN_NAME :URL_BASE :COOKIE_TOKEN_NAME :SESSION_CRYPT_KEY :SESSION_CRYPT_IV :DEVICE_TOKEN_KEY_NAME :TIMEOUT];
    return self;
}

@end
